hack/test-cmd.sh

```diff
@@ -142,7 +142,7 @@ os::cmd::expect_success_and_text "openshift ex validate master-config ${MASTER_C
 os::cmd::expect_success_and_text "openshift ex validate node-config ${NODE_CONFIG_DIR}/node-config.yaml" 'SUCCESS'
 # breaking the config fails the validation check
 cp ${MASTER_CONFIG_DIR}/master-config.yaml ${BASETMPDIR}/master-config-broken.yaml
-os::util::sed '7,12d' ${BASETMPDIR}/master-config-broken.yaml
+echo "kubernetesMasterConfig: {}" >> ${BASETMPDIR}/master-config-broken.yaml
```

Comment:
If you add this line at the end, the config is loaded and fails validation like so:

    start_master.go:271] kubernetesMasterConfig.proxyClientInfo: Invalid value: "": if no client certificate is specified, TLS pods and services cannot validate requests came from the proxy

Is that sufficient or could you suggest something more correct to fail validation as intended?